### PR TITLE
Add latest webpage view

### PR DESCRIPTION
**Pull request by frm**
image

### CODE REVIEW

File: src/popup/components/Protocol/DataScreen/WebpageViews.tsx [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+  );
+}
+
+function LatestWebpage() {
+  const [item, setItem] = useState<StoredItem>();
+
+  useEffect(() => {
+    (async () => {
+      const iterContainer = DataHost.instance().iter();
+      const iter = iterContainer[Symbol.asyncIterator]();
```
Comment (shelbyd) [inline src/popup/components/Protocol/DataScreen/WebpageViews.tsx:171]: This seems like accessing the internals of iterators. We can also access it with an async iterator, which seems more idiomatic, but you can't get just the first item.

File: src/popup/components/Protocol/DataScreen/WebpageViews.tsx [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+            <Icon name={IconNames.VALID} />
+          </ActivationIcon>
+        </HeaderStatus>
+      </Header>
+    </HeaderContainer>
+  );
+}
+
+function LatestWebpageTextValue({ item }: { item?: StoredItem }) {
+  console.log("Item", item);
```
Comment (shelbyd) [inline src/popup/components/Protocol/DataScreen/WebpageViews.tsx]: Remove.

File: src/popup/components/Protocol/DataScreen/WebpageViews.tsx [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+    </span>
+  );
+}
+
+function LatestWebpage() {
+  const [item, setItem] = useState<StoredItem>();
+
+  useEffect(() => {
+    (async () => {
+      const iterContainer = DataHost.instance().iter();
```
Comment (shelbyd) [inline src/popup/components/Protocol/DataScreen/WebpageViews.tsx:170]: We should not merge this until it's correctly using the reverse iterator. Leaving unapproved for now.

**Review by shelbyd — COMMENTED** [review]
Why are you working on this? It's pretty far down in Tracker, unpointed, and unstarted. I'd much prefer you to be working on, for example, showing the private key.

File: src/popup/components/Protocol/DataScreen/WebpageViews.tsx (hunk 1 above)
Comment (frm) [reply src/popup/components/Protocol/DataScreen/WebpageViews.tsx:171]: Yeah, that's the reason I did it like this.

File: src/popup/components/Protocol/DataScreen/WebpageViews.tsx (hunk 3 above)
Comment (frm) [reply src/popup/components/Protocol/DataScreen/WebpageViews.tsx:170]: 👍 agreed

File: src/popup/components/Protocol/DataScreen/WebpageViews.tsx (hunk 2 above)
Comment (frm) [reply src/popup/components/Protocol/DataScreen/WebpageViews.tsx]: Oops, my bad

**Comment by frm** [comment]
> Why are you working on this? It's pretty far down in Tracker, unpointed, and unstarted. I'd much prefer you to be working on, for example, showing the private key.

Already handled that, this screen was being too much of an annoyance for me since we got no design to replace the single and ugly "copy address". I wasn't confident enough to ship that today and risk it, so I took it to add that implementation. It's finished now, as well as some others.

**Review by shelbyd — APPROVED** [review]

**Pull request merged**